src/bridge/IrcHandler.ts

```diff
@@ -406,7 +406,15 @@ export class IrcHandler {
                 virtualMatrixUser.getId()
             ).invite(
                 room.getId(), invitee
-            );
+            ).catch(err => {
+                req.log.warn(
+                    `Failed to invite %s as the inviter user (reason: ${err}),
+                    inviting as a bot as fallback`
+                );
+                // TODO Note the original inviter in the reason
```

Comment:
We could just set some room state here tbh, I don't think we should wait for the MSC to arrive :older_man:

Reply:
What could we set in a room state that'll convey the relevant info?

Reply:
Oh, I mean literally send a m-room-member event with a membership of invite and a reason of whatever.

Reply:
Ah yes, that works. Though currently Synapse seems to just strip the reason anyway, so it doesn't actually show up – but we should at least be future-proof with this :) Done in 2e80b23e.

Reply:
:roll_eyes: synaaaaapse. Is there a bug for that? Otherwise, sounds good.

Reply:
(Seems like it does not strip it after all, all works well, so this is all good).